src/index.js

```diff
@@ -84,6 +87,18 @@ class IOpipeWrapperClass {
     this.timeout = setupTimeoutCapture(this);
 
     this.report = new Report(this);
+
+    try {
+      this.userFuncValue = userFunc.call(
```

Comment:
everything is stored in this anyway, so making a .invoke that returns the value seems a lot cleaner than allocating the return value?

I don't know how nodejs handles return values, if assignments are pointers to the original data, or if it creates copies? I guess it's going to be pointers, but I become cautious when we assign variables to data of quantities and types unknown.